examples/blaze/src/main/scala/com/example/http4s/blaze/demo/server/Server.scala

```diff
@@ -0,0 +1,30 @@
+package com.example.http4s.blaze.demo.server
+
+import cats.effect.{Effect, IO}
+import fs2.StreamApp.ExitCode
+import fs2.{Scheduler, Stream, StreamApp}
+import org.http4s.client.blaze.Http1Client
+import org.http4s.server.blaze.BlazeBuilder
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object Server extends HttpServer[IO]
+
+class HttpServer[F[_]](implicit F: Effect[F]) extends StreamApp[F] {
+
+  override def stream(args: List[String], requestShutdown: F[Unit]): Stream[F, ExitCode] =
+    Scheduler(corePoolSize = 2).flatMap { implicit scheduler =>
+      for {
+        client   <- Stream.eval(Http1Client[F]())
```

Comment:
Http1Client.stream here. It ensures proper resource release.

Reply:
Sure, I wasn't aware of the stream bracketed version of the HttpClient. I'll use it everywhere now!